src/main/plugins/org.dita.eclipsehelp/build_dita2eclipsehelp_template.xml

```diff
@@ -26,14 +29,21 @@ See the accompanying LICENSE file for applicable license.
 	
     <target name="dita2eclipsehelp"
             unless="noMap"
-        depends="build-init, dita.eclipsehelp.init, preprocess, 
+        depends="dita.eclipsehelp.init, build-init, preprocess, 
                  xhtml.topics,
                  copy-css">
       <antcall target="dita.map.eclipse"></antcall>
+      <jar destfile="${output.dir}/${args.eclipsehelp.jar.name}" basedir="${dita.output.dir}" if:set="args.eclipsehelp.jar.name">
```

Comment:
FWIW I originally placed the `<jar>` element in a target of its own. But with the peculiar dependency chain here (5 dependency targets followed by nested `<antcall>`), the only way that works is to move the JAR target into the final spot in the final `<antcall>`. That is ... awkward and sort of hides what is happening, so I moved it in here right after `<antcall>`.

As to why the `<antcall>` is there, I'm not really sure, but I don't see much reason to refactor that at this point.